Jenkinsfile

```diff
@@ -148,19 +122,8 @@ dockerizedBuildPipeline(
   },
   onFailure: {
     githubStatusUpdate('failure')
+    archiveArtifacts(artifacts: 'gallery/test-results/**/*')
```

Comment:
This makes any visual diffs that come up on Jenkins accessible as build artifacts